Настроил проект, добавил доменные сущности, написал каркас для сервисов и репозиториев

WallKing110 opened the pull request.

Commit:
This is project.

Commit:
Hello world!

WallKing110 committed:
Configured project.

WallKing110 committed:
Made form of project.

WallKing110 committed:
Rebase develop on master

IronOxidIII committed:
Merge pull request #5 from IronOxidIII/salamakhin/HomeWork25.09.2025

Salamakhin/home work25.09.2025

WallKing110 committed:
Распределил сущности по полкам, оставил место для творчества

WallKing110 committed:
По ревью добавил удаление юзера в UserService. Исправил опечатку в методе.

IronOxidIII committed:
Merge pull request #6 from IronOxidIII/salamakhin/domains_services_repositories

Распределил сущности по доменам, прописал их, но не до конца(equals, toString). Добавил макеты репозиториев и сервисов.

danone05 committed:
delete all print methods

remove console output and use logging

fix

Refactor: add logging, use getRemindById and List

Заменён вывод через System.out на SLF4J, методы сервиса используют getRemindById,
в репозитории возвращается List вместо ArrayList.

replace int status with RemindStatus enum, add toString(), and update documentation

upd

add-remind-deletion-methods

Methods Reminder

Реализация методов в RemindRepository, RemindService

danone05 committed:
Methods Reminder

Реализация методов в RemindRepository, RemindService

danone05 committed:
add-remind-deletion-methods

danone05 committed:
upd

danone05 committed:
replace int status with RemindStatus enum, add toString(), and update documentation

WallKing110 commented:
закрываю, тк надо еще подумать над тем, как лучше это все оформить в 2 мр-а

lostflydev commented on .idea/misc.xml:
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ExternalStorageConfigurationManager" enabled="true" />
```
папку ./idea закиньте в гитигнор общий

WallKing110 replied:
Добавил в другом мр-е

lostflydev commented on thereminderbot/src/main/java/org/thereminderbot/Main.java:
```diff
@@ -0,0 +1,89 @@
+
+public class Main {
+    public static void main(String[] args) {
+        var userRepository = new UserRepository();
```
В след МР давай все это в компоненты разнесем, как в примере с пары
Коммент можно закрыть после чтения

lostflydev commented on thereminderbot/src/main/java/org/thereminderbot/domain/Remind.java:
```diff
@@ -0,0 +1,144 @@
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+
+/**
```
Я уже видел этот файл в других МР - чтобы такого не было, можно в качестве target ветки ставить ветку еще не закрытого МРа, а не мастер

WallKing110 replied:
Первый блин комом). В следующих мр-ах поумнее буду делать, здесь какой-то бардак получился в итоге

lostflydev commented on thereminderbot/src/main/java/org/thereminderbot/domain/User.java:
```diff
@@ -0,0 +1,98 @@
+    /**
+     * Отличие часового пояса пользователя относительно +0.
+     */
+    private int timeZoneOffset;
```
для этого есть спец типы в джаве - ZoneId

WallKing110 replied:
Спасибо за наводку, изменил в другом мр-е

lostflydev commented on thereminderbot/src/main/java/org/thereminderbot/enums/RemindStatus.java:
```diff
@@ -0,0 +1,16 @@
+/**
+ * Статусы заметки.
+ */
+public enum RemindStatus {
```
за енамы лайк

WallKing110 replied:
Спасибо

danone05 committed:
Refactor: add logging, use getRemindById and List

Заменён вывод через System.out на SLF4J, методы сервиса используют getRemindById,
в репозитории возвращается List вместо ArrayList.

WallKing110 committed:
Правки по ревью.

Заменил тип для часового пояса и добавил .idea в гитигнор. Тест коммента к коммиту.

danone05 committed:
fix

danone05 committed:
remove console output and use logging

danone05 committed:
delete all print methods

danone05 committed:
Merge branch 'Remind' of https://github.com/IronOxidIII/TRB_TheReminderBot into Kroshakov/implement-remind-repository-and-service-methods

WallKing110 committed:
Добавил ямль файл

WallKing110 committed:
Добавил site-plugin и distributionManagement для maven-delpoy

WallKing110 committed:
Добавил depgraph, может заработает

WallKing110 committed:
Удалил depgraph из workflows(

WallKing110 committed:
Проверка билда, специально добавил ошибку.

WallKing110 committed:
Настроил билд для maven, указал src и test.

В следствие этого, теперь проект настраивается из pom.xml, из-за этого кодировка у файлов сменилась.

WallKing110 committed:
Добавил ошибку, для проверки билда.

WallKing110 committed:
Убрал ошибку для проверки билда

WallKing110 committed:
Мелкие правки в форматировании.

WallKing110 committed:
Merge pull request #10 from IronOxidIII/salamakhin/domains_services_repositories

Правки по ревью из мр-а с доменами.

WallKing110 committed:
Merge branch 'develop' into salamakhin/add_ci

WallKing110 committed:
Merge pull request #12 from IronOxidIII/salamakhin/add_ci

Добавил CI с билдом.

WallKing110 committed:
Merge pull request #7 from IronOxidIII/Remind

kroshakov/implement-remind-repository-and-service-methods

IronOxidIII committed:
Sesekin/implementation_for_UserRepository_and_UserService (#8)

* Done!

* Переделано всё кроме private RemindRepository.

Переделано всё кроме private RemindRepository

* Внесены правки

* Внесены правки

* Внесены правки

* Внесены правки

* Исправлен билд после правок.

---------

Co-authored-by: SAN <a.p.3325@yandex.ru>
Co-authored-by: WallKing110 <57759414+WallKing110@users.noreply.github.com>
Co-authored-by: WallKing110 <ksalamahin2004@mail.ru>

IronOxidIII committed:
#15 Модификация ci

IronOxidIII committed:
Merge pull request #25 from IronOxidIII/Sesekin/ci_addition

#15 Модификация ci
